src/raw_stream.rs

```diff
@@ -159,7 +159,7 @@ impl RawDevice {
         Self::from_fd(fd)
     }
 
-    fn from_fd(fd: OwnedFd) -> io::Result<RawDevice> {
+    pub fn from_fd(fd: OwnedFd) -> io::Result<RawDevice> {
```

Comment:
Ah, forgot to ask, but can you PR in some rustdoc now that this is `pub`?

Reply:
thanks for the quick merge, docs filed as #137 now, is that what you had in mind?